job-descriptions/software-engineer-intern.md

```diff
@@ -16,17 +16,17 @@ Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have be
 Our small team consists of talented, mature, collaborative, driven individuals who are attracted to the massive problem we are tackling. We work in an open environment that treats people in a first-class manner and provides them with ownership, responsibility, and autonomy.
 
 ### About the role
+*This is a paid internship* 
 
 As an intern, you will help build Sourcegraph, a multi-tier application (web, CLI, browser extensions, API, data stores, services) written primarily in TypeScript (frontend) and Go (backend). You will be matched with a mentor who will help plan your work, and then you will have responsibility and autonomy to get the work done.
 
-Requirements:
-- You are passionate about building tools for developers.
 
 Ideal candidates:
 
 - Are pursuing a computer science undergraduate or graduate degree.
-- Are familiar with one or more of the technologies that we use (e.g. Go, TypeScript, React).
-- Have built or contributed to a developer tool or library (e.g. https://github.com/sourcegraph/sourcegraph).
+- Are passionate about developer tools and have worked in the developer tool space in some capacity (built a developer tool, wrote a blog post, recorded a podcast, etc.)
+- Are active in the open ource community.
```

Comment:
Missing an "s"

Reply:
Also, let's remove this bullet, and instead put it in the bullet above.

The one above could read:

- Can demonstrate passion and excitement about developer tools (e.g. have built a developer tool, contributed to open source developer tools or other open source projects, written blog posts or talks about development, etc.)